zebra-consensus/Cargo.toml

```diff
@@ -36,6 +36,7 @@ wagyu-zcash-parameters = "0.2.0"
 
 [dev-dependencies]
 color-eyre = "0.5.11"
+rand07 = { package = "rand", version = "0.7" }
```

Comment:
It seems like we're using rand 0.4, 0.5, 0.6, and 0.7 in different dependencies.

That's ok for now, but we should clean it up eventually.
Can you open a ticket to upgrade as many as possible?
It should be blocked by the tokio upgrade #2200

Reply:
Sure! I opened #2391 for this :+1: